portal/src/texts/profile/picture.md

```diff
@@ -5,303 +5,59 @@ order: 8
 
 ### Skadeforsikring
-Vi viser folk og tingene deres i samspill, heller enn å ta bilde av et produkt. (Viser vi skader?)
+Vi viser folk og tingene deres i samspill, heller enn å ta bilde av et produkt.
 
 ### Bilforsikring
-Vi tar bilder i og rundt bilen. I trafikken, ved av- og pålessing, av folk og ting i nærheten av bilen, eller når de steller bilen sin. Kan hende pakker de til ferie, henter på trening, er på handletur eller vasker bilen. Situasjonen er viktigere enn bilen, bildet skal ikke oppfattes som reklame for en bestemt biltype. Vær oppmerksom på at bilen skal kjøre på norske veier.
-
-**bilde: i bilen + ved bilen**
+Bruk bilder av situasjoner i og rundt bilen. I trafikken, ved av- og pålessing, av folk og ting i nærheten av bilen, eller når de steller bilen sin. Kan hende pakker de til ferie, henter på trening, er på handletur eller vasker bilen. Situasjonen er viktigere enn bilen, bildet skal ikke oppfattes som reklame for en bestemt biltype. Vær oppmerksom på at bilen skal kjøre på norske veier.
 
 ### Reiseforsikring
 Når det gjelder reiseforsikring kan vi bruke bilder fra utlandet, men da gjerne i typiske situasjoner man havner i som turist. Her kan vi appellere til ulike typer, for eksempel den eventyrlystne, storbyturisten, solslikkeren eller den lokale turisten.
 
 Merk: Fremtind er et bærekraftig selskap og vi unngår å bruke bilder av fly, cruiseskip eller aktiviteter som ødelegger natur eller lokalsamfunn.
 
-**bilde: generell turist + bilde av sted/reisetyper vi ikke “reklamerer” for**
-
 ### Dyreforsikring
 Dyr har sin egen personlighet og de kan gjerne eie bildet alene, men det er også fint å vise samspillet mellom dyr og mennesker. Dyr egner seg fint til bilder tatt “rett ovenfra”.
 
-**bilde: dyr ovenfra**
-
--   Skreddersy bilder til de ulike forsikringsproduktene.
--   (Viser vi skade?)
--   Ta bilder rundt og i bilen. uten at det ser ut som en bilreklame.
--   Bruk kjente turistsituasjoner for reiseforsikring.
--   Bruk dyr alene eller dyr og folk sammen til dyreforsikring.
-
 ## Lys og farger
-
 Vi vil at atmosfæren i bildene skal virke inviterende og virkelighetsnær. For å skape det bruker vi dagslys, eller naturlig lyssetting og fargekorrigering.
 
-**bilde: prakteksemplar**
-
-Unngå skarpt lys og harde skygger, spesielt i ansikter.
-
-**bilde: skygger i ansikt, bilde der hovedmotivet forsvinner i skygge**
-
-Unngå hvit bakgrunn som skaper høy kontrast og skarpe kanter. Det oppstår ofte der det er hvitmalte vegger, eller ute når himmelen er lys.
-
-**bilde: bilde med overeksponert himmel + riktig eksponering eller annet vær + innebilde med ulik veggfarge**
-
 Vi bruker ikke effekter, filtre eller for mye refleksjon (lens flare) på bildene våre. Alle bildene i Fremtinds bildedatabase er redigert og fargekorrigert, slik at de oppfyller kravene til bildestilen. De skal oppleves som like.
```

Comment:
Tenker det holder å si effekter og filtre og fjerne «for mye refleksjon (lens flare)». Refleksjon er ikke nødvendigvis et problem, men kunstig lens flare, og det går under effekter.